### DISCUSSION TITLE
Feature Request: Add `EXCLUDE` clauses to window functions

### DISCUSSION THREAD

**Discussion by j-w-moebius** [duckdb/duckdb#7829] [Ideas]
I noticed that the use of `EXCLUDE` clauses in window functions is currently not supported.
See https://www.postgresql.org/docs/current/sql-select.html under __frame\_exclusion__ for what this could look like. 
Example query:
```
SELECT array_agg(i) over win
FROM generate_series(1,9) AS _(i)
WINDOW win  AS (ORDER BY i ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING EXCLUDE CURRENT ROW);


 array_agg 
-----------
 [2]
 [1,3]
 [2,4]
 [3,5]
 [4,6]
 [5,7]
 [6,8]
 [7,9]
 [8]
```
Is anyone from the core team currently working or planning to work on this? If not, I'll try to implement this.

**Comment by hawkfish** [comment]
Implemented.
